tts-service/web/get.go

```diff
@@ -5,7 +5,7 @@ import (
 	"net/http"
 )
 
-func onGetByIdRequest(h getHandling, w http.ResponseWriter, r *http.Request) {
+func onGetByIdRequest(h getOrDeleteHandling, w http.ResponseWriter, r *http.Request) {
```

Comment:
Try to avoid ThisOrThat types/functions in your programs. This is VERY bad style. Each language gives you means of handling that. In golang you can easily create type aliases, and I'd use that.
Yes, it's two (or four) more LOC, but way more readable.
Example:
type ByIdHandling struct { ... } //generic
type GetByIdHandling ByIdHandling //alias, behaves the same.
type DeleteByIdHandling ByIdHandling //alias